@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -31,12 +31,18 @@ function overlayObject(object: TileObject, displacementMap: THREE.DataTexture):
 
     if (Array.isArray(material)) {
         material.forEach(mat => {
-            if (hasDisplacementFeature(mat)) {
+            if (hasDisplacementFeature(mat) && mat.displacementMap !== displacementMap) {
                 mat.displacementMap = displacementMap;
+                mat.needsUpdate = true;
             }
         });
-    } else if (material && hasDisplacementFeature(material)) {
+    } else if (
+        material &&
+        hasDisplacementFeature(material) &&
+        material.displacementMap !== displacementMap
+    ) {
         material.displacementMap = displacementMap;
+        material.needsUpdate = true;
```

Comment:
Same here, Missing setting the needsUpdate flag for the displacement texture. 
You missed setting the flags in setupTerrainMaterial() in TileGeometryCreator, next to the line doing:
```
stdMaterial.displacementMap = null;
```
Maybe it makes sense to move out to a function.

Reply:
I fixed this one as well.